yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -319,6 +319,16 @@ package object config {
     .stringConf
     .createOptional
 
+  private[spark] val CREDENTIALS_RENEWAL_TIME = ConfigBuilder("spark.yarn.credentials.renewal")
+    .internal()
+    .timeConf(TimeUnit.MILLISECONDS)
+    .createWithDefault(Long.MaxValue)
+
+  private[spark] val CREDENTIALS_UPDATE_TIME = ConfigBuilder("spark.yarn.credentials.update")
```

Comment:
so this is really executors fetching credentials from AM correct?  It took me a few minutes to figure out what this config was vs the one above, can we rename it  to spark.yarn.credentials.executors.updateTime and CREDENTIALS_EXECUTOR_UPDATE_TIME.